.github/workflows/android.yml

```diff
@@ -0,0 +1,73 @@
+name: Android
+
+on:
+   push:
+      branches: [ RC_2_0 master ]
+   pull_request:
+
+env:
+  BOOST_DOT_VERSION: "1.75.0"
+  BOOST_VERSION: "1_75_0"
+  NDK_VERSION: "r21d"
+  OPENSSL_VERSION: "1.1.1i"
+  OPENSSL_NO_OPTS: "no-deprecated no-shared no-makedepend"
+
+jobs:
+
+   android_arm_build:
+      name: Build Android Arm 32bits
+      runs-on: ubuntu-20.04
+
+      steps:
+      - name: checkout
+        uses: actions/checkout@v2.3.3
+        with:
+           submodules: true
+
+      - name: install boost
+        run: |
+          wget -nv -O boost.tar.gz https://dl.bintray.com/boostorg/release/${BOOST_DOT_VERSION}/source/boost_${BOOST_VERSION}.tar.gz
```

Comment:
I had an issue with this earlier, when doing this on travis. bintray has a quota for boost and the download would start to fail until the quota was replenished. iirc, that's when I changed to download from github instead.

Reply:
I remember, also, there is some noise about Bintray going down: https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

I think I can use the one from the system, will try it.

Reply:
if only there was some way to crowd-source bandwidth... :)